chart/orkestra/values.yaml

```diff
@@ -62,14 +62,14 @@ debug:
 
 # Dependency overlay values
 chartmuseum:
+  interval: 5m
   env:
     open:
       DISABLE_API: false 
 
 helm-operator:
```

Comment:
We can get rid of this entirely

Reply:
Get rid of the entire section labeled chartmuseum or just the interval portion?

Reply:
Ah I meant helm-operator 🤦